docs/how_to_use_epsilon.rst

```diff
@@ -0,0 +1,67 @@
+
+Combining HDBSCAN\* with DBSCAN 
+=============================
+
+While DBSCAN needs a minimum cluster size *and* a distance threshold epsilon as user-defined input parameters, 
+HDBSCAN\* is basically a DBSCAN implementation for varying epsilon values and therefore only needs the minimum cluster size as single input parameter.
+The ``'eom'`` (Excess of Mass) cluster selection method then returns clusters with the best stability over epsilon.
+
+Unlike DBSCAN, this allows to find clusters of variable densities without having to choose a suitable distance treshold first.
+However, there are cases where we could still benefit from the use of an epsilon threshold.
+
+For illustration, see this map with GPS locations, representing recorded pick-up and drop-off locations for customers of a ride pooling provider.
+The largest (visual) data cluster can be found around the train station. Smaller clusters are placed along the streets, depending on the requested location
+in form of a postal address or point of interest. Since we are considering a door-to-door system where customers are not bound to collective pick-up or
```

Comment:
```suggestion
in the form of a postal address or point of interest. Since we are considering a door-to-door system where customers are not bound to collective pick-up or
```